### DISCUSSION TITLE
How to use automations?

### DISCUSSION THREAD

**Discussion by bodagrave** [TandoorRecipes/recipes#1495] [Q&A] [answered]
This function really is a burner, for it saves a lot of time.
Unfortunately, I have not really an idea how to use it properly. Why do I have three params?
As far as I understood using first 2 params would lead into something like this:

param 1: fat reduced milk
param 2: Milk

Would translate "fat reduced milk" into "milk", right?
But what do I do with the third one?

Some hint really would be appreciated!

Stay healthy and hungry :-)

Boda

**Comment by vabene1111** [comment, marked as answer]
It is not documented yet on purpose as its in a really early state of beta where we are just playing around with it. 

You should not create automations manually but only via the merge function in unit/food views and the merges are then automatically applied when importing a recipe. Thats it for now. I will need to properly review the implementation but havn't gotten around to doing this. Once thats done we can start making this feature more user firendly, documenting it and also persisting it for the long term future.

**Comment by bodagrave** [comment]
Hi vabene,
that is good to know. I'm keen to the the progress here, and even though my programing skills suck, I for sure still can help with some of the translations, as soon as there is some docs available.

**Reply by vabene1111** [reply]
thanks :) you can find everything you need to know regarding translations here https://docs.tandoor.dev/contribute/#contribute-translations
